Integrations/integration-opswat-metadefender.yml

```diff
@@ -0,0 +1,239 @@
+description: At the heart of the solution, the Metadefender multi-scanning engine
+  uses 30+ anti-malware engines to scan files for threats, significantly increasing
+  malware detection.
+configuration:
+- display: Server URL (e.g. https://api.metadefender.com/v2)
+  name: url
+  defaultvalue: https://api.metadefender.com/v2
+  type: 0
+  required: true
+- display: API Key
+  name: api
+  defaultvalue: ""
+  type: 4
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    var sendRequest = function(url,param, fileId) {
+        var requestUrl = params.url + url + param;
+        var res;
+
+        if (fileId) {
+            var fileName = dq(invContext, 'File(val.EntryID == \'' + fileId + '\').Name');
```

Comment:
again check it exists first

Reply:
What do you mean? I am checking:
" if (fileId) {"

Reply:
meant the `dq` result